bin/npm-ci-publish.js

```diff
@@ -39,47 +39,53 @@ let pkg = readJsonFile('package.json');
 const previousVersion = pkg.version;
 const shouldUnlink = false;
 
-execCommandAsync('npm run release --if-present').then(({ stdio }) => {
-  if (shouldUnlink) {
-    unlinkSync('.npmrc');
-  }
-  pkg = readJsonFile('package.json');
-  if (
-    pkg.scripts &&
-    pkg.scripts.release &&
-    pkg.scripts.release.indexOf('mltci') > -1
-  ) {
-    console.log(
-      'Detected release mltci, it might publish, so skipping publishing myself',
-    );
-  } else if (pkg.private && previousVersion !== pkg.version) {
-    console.log('forcing republish in order to sync versions');
-    delete pkg.private;
-    writeJsonFile('package.json', pkg);
-  }
+Promise.resolve()
```

Comment:
The only change happened here, wrapping the `run release` in an `if`. The other changes are caused by indentation.